src/Exception/NotFoundException.php

```diff
@@ -4,6 +4,6 @@
 
 class NotFoundException
     extends \InvalidArgumentException
-    implements \Interop\Container\Exception\NotFoundException
+    implements \Psr\Container\NotFoundExceptionInterface
```

Comment:
There is some inconsistency in the coding style. Some make use of `use` and some not. Just my thoughts.